Tests/GRPCTests/CallPathTests.swift

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
This whole file was copied from the routing handler tests (which is now deleted)